<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ilista</class>
 <widget class="QWidget" name="Ilista">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(228, 235, 240);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; color:#ff0000;&quot;&gt;Lista Duplamente Encadeada&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; color:#ff0000;&quot;&gt;INICIO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; color:#ff0000;&quot;&gt;FIM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; color:#ff0000;&quot;&gt;POSIÇÃO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInserirInicio">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>INSERIR INICIO</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Área de Trabalho/Estrutura de Dados/download.png</normaloff>../../Área de Trabalho/Estrutura de Dados/download.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAcessarInicio">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACESSAR INICIO</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Área de Trabalho/Estrutura de Dados/download.png</normaloff>../../Área de Trabalho/Estrutura de Dados/download.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRetirarInicio">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>RETIRAR INICIO</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Área de Trabalho/Estrutura de Dados/download.png</normaloff>../../Área de Trabalho/Estrutura de Dados/download.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInserirFim">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>INSERIR FIM</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Área de Trabalho/Estrutura de Dados/upload.png</normaloff>../../Área de Trabalho/Estrutura de Dados/upload.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAcessarFim">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACESSAR FIM</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Área de Trabalho/Estrutura de Dados/upload.png</normaloff>../../Área de Trabalho/Estrutura de Dados/upload.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRetirarFim">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>RETIRAR FIM</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Área de Trabalho/Estrutura de Dados/upload.png</normaloff>../../Área de Trabalho/Estrutura de Dados/upload.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPosicaoInicial">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>INSERIR POSICAO</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Área de Trabalho/Estrutura de Dados/POSICAO.png</normaloff>../../Área de Trabalho/Estrutura de Dados/POSICAO.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRetirarPosicao">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>240</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>RETIRAR POSIÇÃO</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Área de Trabalho/Estrutura de Dados/POSICAO.png</normaloff>../../Área de Trabalho/Estrutura de Dados/POSICAO.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInerirPosicao">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>170</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACESSAR POSIÇÃO</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Área de Trabalho/Estrutura de Dados/POSICAO.png</normaloff>../../Área de Trabalho/Estrutura de Dados/POSICAO.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditSaida">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>991</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>300</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOrdenar">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ORDENAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ff0000;&quot;&gt;MOSTRAR LISTA :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(228, 235, 240);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Quantidade de Elementos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQE">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(228, 235, 240);</string>
   </property>
   <property name="text">
    <string>Valor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditValor">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>370</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 235, 240);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(228, 235, 240);</string>
   </property>
   <property name="text">
    <string>Posicao</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>370</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Área de Trabalho/Estrutura de Dados/horizontal-210x130.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>0</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Área de Trabalho/Estrutura de Dados/SA.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-20</y>
     <width>1041</width>
     <height>681</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Área de Trabalho/Estrutura de Dados/fundoto.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_11</zorder>
  <zorder>pushButtonInserirInicio</zorder>
  <zorder>pushButtonAcessarInicio</zorder>
  <zorder>pushButtonRetirarInicio</zorder>
  <zorder>pushButtonInserirFim</zorder>
  <zorder>pushButtonAcessarFim</zorder>
  <zorder>pushButtonRetirarFim</zorder>
  <zorder>pushButtonPosicaoInicial</zorder>
  <zorder>pushButtonRetirarPosicao</zorder>
  <zorder>pushButtonInerirPosicao</zorder>
  <zorder>textEditSaida</zorder>
  <zorder>comboBox</zorder>
  <zorder>pushButtonOrdenar</zorder>
  <zorder>label_6</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEditQE</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEditValor</zorder>
  <zorder>label_10</zorder>
  <zorder>spinBox</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
